VideoAPI/VideoApi.Events/Handlers/MediaProblemEventhandler.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR;
+using VideoApi.DAL.Commands;
+using VideoApi.DAL.Commands.Core;
+using VideoApi.DAL.Queries;
+using VideoApi.DAL.Queries.Core;
+using VideoApi.Domain.Enums;
+using VideoApi.Events.Handlers.Core;
+using VideoApi.Events.Hub;
+using VideoApi.Events.Models;
+using VideoApi.Events.ServiceBus;
+using TaskStatus = VideoApi.Domain.Enums.TaskStatus;
+
+namespace VideoApi.Events.Handlers
+{
+    public class MediaProblemEventHandler : EventHandlerBase
```

Comment:
Rename to the event type name discussed above

Reply:
Updated the handler name